capellambse_context_diagrams/__init__.py

```diff
@@ -272,3 +273,38 @@ def register_data_flow_view() -> None:
     for class_, dgcls, default_render_params in supported_classes:
         accessor = context.DataFlowAccessor(dgcls.value, default_render_params)
         common.set_accessor(class_, "data_flow_view", accessor)
+
+
+def register_element_relation_view() -> None:
+
+    supported_classes = [
+        (
+            oa.CommunicationMean,
+            {
+                DiagramType.OAB,
+            },
+        ),
+        (
+            fa.ComponentExchange,
+            {
+                DiagramType.SAB,
+                DiagramType.LAB,
+                DiagramType.PAB,
+            },
+        ),
+    ]
+
+    styles: dict[str, dict[str, capstyle.CSSdef]] = {}
+    for _class, _diagclasses in supported_classes:
+        common.set_accessor(
+            _class,
+            "element_relation_view",
+            context.ElementRelationAccessor("ElementRelationView Diagram"),
+        )
```

Comment:
It is different from the usual initializing:

```suggestiondef register_element_relation_view() -> None:
    supported_classes: list[SupportedClass] = [
        (oa.CommunicationMean, {DiagramType.OAB}, {}),
        (
            fa.ComponentExchange,
            {DiagramType.SAB, DiagramType.LAB, DiagramType.PAB},
            {},
        ),
    ]

    styles: dict[str, dict[str, capstyle.CSSdef]] = {}
    for _class, _diagclasses, render_params in supported_classes:
        common.set_accessor(
            _class,
            "element_relation_view",
            context.ElementRelationAccessor(
                "ElementRelationView Diagram", render_params
            ),
        )
```